apps/web/next.config.js

```diff
@@ -56,21 +56,41 @@ const baseXYZDomains = 'https://base.mirror.xyz';
 const greenhouseDomains = 'https://boards.greenhouse.io';
 const ccaDomain = 'https://static-assets.coinbase.com/js/cca/v0.0.1.js';
 const ccaLiteDomains = 'https://cca-lite.coinbase.com';
-const sprigDomains = 'https://api.sprig.com https://cdn.sprig.com;';
+const sprigDomains = 'https://api.sprig.com https://cdn.sprig.com';
+const walletconnectDomains =
+  'https://*.walletconnect.org wss://*.walletconnect.org wss://*.walletconnect.com https://*.walletconnect.com https://explorer-api.walletconnect.com';
 
 const contentSecurityPolicy = {
   'default-src': [
     "'self'",
     "'unsafe-inline'", // NextJS requires 'unsafe-inline'
     isLocalDevelopment ? "'unsafe-eval'" : '',
     baseXYZDomains,
-    greenhouseDomains,
```

Comment:
does this break greenhouse? iirc we need to embed some things

Reply:
Based on my testing it seems sufficient to have it `connect-src` ie jobs still show up with this config